joomla/joomla-cms
# [4.0] Namespace User - Profile plugin

Pull Request for Issue # .

### Summary of Changes

Namespaces User - Profile plugin.

### Testing Instructions

Delete `libraries/autoload_psr4.php` file.
Enable User Registration.
Enable User - Profile plugin.
In plugin options select a Terms of Service article.
Try to register.
Check that terms of service and date of birth fields appear correctly in the form.

### Documentation Changes Required

IDK.
